styles/components/_misc.scss

```diff
@@ -99,3 +99,8 @@
     opacity: .35;
   }
 }
+
+// This could probably go into an animations components file once we have a couple
+.fadeIn {
```

Comment:
This isn't being used. I was experimenting with having the image fade in on load, but there are some complications on that since it will re-fade in on re-renders. Also, a re-render mid animation doesn't look quite right.

But... leaving the fadeIn animation class in since it could be useful.